Agents/ForecastingAgent/forecasting/datamodel/data_mapping.py

```diff
@@ -0,0 +1,95 @@
+
```

Comment:
Please provide some introduction what this script is used for. This file is referenced quite heavily in the README and people need to make changes here, so please help them ;-)